internal/pkg/utils/strhelper/strhelper.go

```diff
@@ -127,3 +128,7 @@ func FirstN(str string, n int) string {
 	}
 	return str[0:n]
 }
+
+func StripMarkdown(str string) string {
+	return stripmd.Strip(str)
+}
```

Comment:
Na to by sa mohol napisat drobny unit test, aj ked to pokryva test vo validatore (optional).